src/store/tests/tests.selector.ts

```diff
@@ -121,6 +121,13 @@ export const getActivityCodeBySlotId = (testsModel: TestsModel, id: number): Act
   return null;
 };
 
+export const getPassCertificateBySlotId = (testsModel: TestsModel, id: number): String => {
+  if (testsModel && testsModel.startedTests && testsModel.startedTests[id]) {
+    return testsModel.startedTests[id].passCompletion.passCertificateNumber;
+  }
+  return null;
+};
+
```

Comment:
Return type should be lowercase i.e. ```string```;

Pass completion is an optional key, think we need to handle it as such using lodash maybe?

```return get(testsModel.startedTests[id], 'passCompletion.passCertificateNumber', null);```